### ISSUE TITLE
Training data size

### ISSUE THREAD

**Issue by Asanebou** [torralba-lab/im2recipe-Pytorch#17]
Hi I'm trying to process the data from scratch. But I don't know why the training data size is 238459 instead of 720639 as claimed in the CVPR paper. (I also ran "python tokenize_instructions.py train" and got the output file "tokenized_instructions_train.txt" of 720639 lines.) 
I'd appreciate it if you could tell me why and how you chose the 238459 samples. Thank you.

**Comment by nhynes** [comment]
How did you get those 238k data points? I think that you might be using the training or validation set. The 720k is the full dataset.

**Comment by Asanebou** [comment]
@nhynes The 238k data is indeed the training data. I finally find that in "./scripts/mk_dataset.py" , the lines 104~105
` if ninstrs >= opts.maxlen or ningrs >= opts.maxlen or ningrs == 0 or not imgs or remove_ids.get(entry['id']):
        continue`
 filter out the data points that don't meet the conditions, which reduces the training data size from 72k to 23k.
